concent_api/core/templates/admin/payment_report.html

```diff
@@ -0,0 +1,155 @@
+            <td>
+            {% for force_payment_committed in pair_report.pending_responses.force_payment_committed %}
+                <p><a href="{% url 'admin:core_pendingresponse_change' force_payment_committed.id %}">
+                    - delivered: {{ force_payment_committed.delivered }},
+                    created_at: {{ force_payment_committed.created_at }}</a>
+                </p>
+            {% endfor %}
+            </td>
+        </tr>
+
+        {#  Payments table  #}
+        <tr>
+            <th rowspan="5">Payments</th>
+            <th>Regular payments</th>
+            <td>
+            {% for payment in pair_report.payments.regular_payments %}
+                <p>
+                    <a href="https://{{ eth_network }}etherscan.io/address/{{ payment.provider }}">- provider: {{ payment.provider }}</a>,
+                    <a href="https://{{ eth_network }}etherscan.io/address/{{ payment.requestor }}">requestor: {{ payment.requestor }}</a>,
+                    amount: {{ payment.amount }},
+                    <a href="https://{{ eth_network }}etherscan.io/tx/{{ payment.tx_hash }}">tx_hash: {{ payment.tx_hash }}</a>
+                </p>
+            {% endfor %}
+            </td>
+        </tr>
+        <tr>
+            <th>settlement_payments</th>
+            <td>
+            {% for payment in pair_report.payments.settlement_payments %}
+                <p>
+                    <a href="https://{{ eth_network }}etherscan.io/address/{{ payment.provider }}">- provider: {{ payment.provider }}</a>,
+                    <a href="https://{{ eth_network }}etherscan.io/address/{{ payment.requestor }}">requestor: {{ payment.requestor }}</a>,
+                    amount: {{ payment.amount }},
+                    <a href="https://{{ eth_network }}etherscan.io/tx/{{ payment.tx_hash }}">tx_hash: {{ payment.tx_hash }}</a>
+                </p>
+            {% endfor %}
+            </td>
+        </tr>
+        <tr>
+            <th>forced_subtask_payments</th>
+            <td>
+            {% for payment in pair_report.payments.forced_subtask_payments %}
+                <p>
+                    <a href="https://{{ eth_network }}etherscan.io/address/{{ payment.provider }}">- provider: {{ payment.provider }}</a>,
+                    <a href="https://{{ eth_network }}etherscan.io/address/{{ payment.requestor }}">requestor: {{ payment.requestor }}</a>,
+                    amount: {{ payment.amount }},
+                    <a href="https://{{ eth_network }}etherscan.io/tx/{{ payment.tx_hash }}">tx_hash: {{ payment.tx_hash }}</a>
+                </p>
+            {% endfor %}
+            </td>
+        </tr>
+        <tr>
+            <th>additional_verification_payments_for_requestor</th>
+            <td>
+            {% for payment in pair_report.payments.additional_verification_payments_for_requestor %}
+                <p>
+                - subtask_id: {{ payment.subtask_id }},
```

Comment:
Same here, partial template and include.

Reply:
Done